webapp/src/js/services/session.js

```diff
@@ -41,7 +41,11 @@ var COOKIE_NAME = 'userCtx',
       };
 
       var logout = function() {
-        $http.delete('/_session').finally(navigateToLogin);
+        $http.delete('/_session')
+          .catch(function() {
+            // Ignore exception. User can already be logged out.
+          })
+          .finally(navigateToLogin);
```

Comment:
Seeing you're catching the exception this should just be a `then` not a `finally`.